src/ngAnimate/ngAnimateSwap.js

```diff
@@ -10,11 +10,17 @@
  *
  * ngAnimateSwap is a animation-oriented directive that allows for the container to
  * be removed and entered in whenever the associated expression changes. A
- * common usecase for this directive is a rotating banner component which
+ * common usecase for this directive is a rotating banner or slider component which
  * contains one image being present at a time. When the active image changes
  * then the old image will perform a `leave` animation and the new element
  * will be inserted via an `enter` animation.
  *
+ * @animations
+ * | Animation                        | Occurs                               |
+ * |----------------------------------|--------------------------------------|
+ * | {@link ng.$animate#enter enter}  | when the expression changes, on the new element |
+ * | {@link ng.$animate#leave leave}  | when the expression changes, on the old element |
```

Comment:
Here too, I would change the description to something like: _when the new/old element is inserted to/removed from the DOM_